core/types/block.go

```diff
@@ -830,7 +837,11 @@ func (rb *RawBody) DecodeRLP(s *rlp.Stream) error {
 	}
 	var tx []byte
 	for tx, err = s.Raw(); err == nil; tx, err = s.Raw() {
-		rb.Transactions = append(rb.Transactions, tx)
+		_, txContent, _, err := rlp.Split(tx)
```

Comment:
With this change, the `Transactions` in the `RawBody` now contains transaction payloads without size prefixes, right?

Reply:
That's the one yes :) when encoded they are stored as a List with a size prefix, so now I use the split function to get at the data within the list and drop the size prefix.  There's a unit test covering this off in the commit as well for some peace of mind.